gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.1.1-bin.zip
```

Comment:
Bumping the Gradle version should probably not be paired with other code changes. \
I think it is ok for now but we should do that seperated in the future.

Also, why does the gradle-wrapper.jar not change if you change the version?

Reply:
I updated the gradle version directly on the dev branch.